src/server.js

```diff
@@ -365,7 +379,16 @@ module.exports = function (classes) {
         }
 
         this.authHandler = handler;
+      },
+      enableJWTAuth: function (handler) {
+        /*if (!_.isFunction(handler)) {
```

Comment:
if the handler is not a function, we should throw an exception on setting up the authorization handler, so the server does not boot up with some jibberish handler such as a string.